### DISCUSSION TITLE
change WYSIWYG editor option

### DISCUSSION THREAD

**Discussion by altezza04** [directus/directus#5664] [Archive]
I know Directus currently use TinyMCE, but how / where to change the option for it? eg, i am trying to add some context menu in the editor. i need to have access to tinymce.init({}) function

**Comment by rijkvanzanten** [comment]
The WYSIWYG interface has a "TinyMCE Overrides" option you can use to add in any configuration that's passed to `init` 👍🏻

**Reply by altezza04** [reply]
Thanks, but how about a global option overrides? because I have many WYSIWYG field in my setup

**Reply by rijkvanzanten** [reply]
There's no global tinymce override currently

**Reply by Joonel** [reply]
So I can't do this? Is there some other option or approach so I can insert complex html markup in a field?
```
tinymce.PluginManager.add('proscons', function(editor, url) {
    editor.ui.registry.addButton('proscons', {
        text: 'Add Pros/Cons',
        onAction: function() {
            editor.insertContent(
                '*** my custom content ***'
            );
        }
    });
});

```

**Reply by rijkvanzanten** [reply]
@Joonel No, that's not currently possible. Another option would be to make your own wysiwyg interface that does whatever you need it to do https://docs.directus.io/extensions/interfaces.html

**Reply by Joonel** [reply]
Thank you.

**Comment by sacconazzo** [comment]
Hi, I was wondering if there was any news about that in directus version 9:
To have a default configuration valid for all wysiwyg fields of the project.

In this way, if we change the default configuration, all the fields of each collection would inherit the changes.
